README.md

```diff
@@ -262,6 +262,8 @@ func main() {
 	// Basic one
 	if x > 0 {
 		return x
+	} else if x == 10 {
+		return 10
```

Comment:
A good point however it can never go to this line. x is 10 meets x > 0.

Reply:
Correct - did it in a hurry and was a bit tired